docs/howto/webhook_receiver.md

```diff
@@ -0,0 +1,335 @@
+# How to write a webhook receiver
+
+This guide shows how to add a receiver using the webhook technique.
+
+---
+
+**Note:** You have to check the input source docs to implement most of the receiver's methods.
+
+---
+
+## Choose an input source
+
+The input source is the main platform we use when publishing new content to our live blog. Therefore, we must first decide on the input source to use.
+
+Some examples of possible input sources are [Facebook's Messenger Platform](https://developers.facebook.com/docs/messenger-platform), [Whatsapp](https://developers.facebook.com/docs/whatsapp), [Zulip](https://zulip.com/api/).
+
+The only requirement is that it should allow setting up an outgoing webhook connection.
+
+## Handle webhook connection
+
+After choosing an input source, we need to handle webhook connections with it. 
+
+The [`WebhookReceiverMixin`](../reference/receivers/webhook_receiver_mixin.md) is designed for that purpose. It makes the following assumptions about the input source:
+
+- It sends new updates using POST requests,
+- It expects to receive an HttpResponse `OK` after sending our site a new update.
+
+
+### Define a `webhook url`
+
+First, let's define the `webhook_url` that the input source uses to send us new updates.
+
+The base path of this url is `/wagtail_live/`. We need to specify an additional path parameter that is specific to an input source.
```

Comment:
> The base path of this url is `/wagtail_live/`.

This assumes the same url path as in the tutorial was chosen right? Should we make that clear?

Reply:
I don't understand what you mean.